### PR TITLE
Added routes and binary styling to DP2P

### PR DESCRIPTION
**Pull request by mahdiaryayi-fs**

### CODE REVIEW

File: scripts/config/pages.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = [
     ['cashier/payment_agent_listws',            'app/cashier/payment_agent_list',              'default',    'Payment Agent Deposit'],
     ['cashier/payment_methods',                 'app/cashier/payment_methods',                 'default',    'Payment Methods'],
     ['cashier/session_expired',                 'app/cashier/session_expired',                 null,         'Cashier session expired'],
+    ['cashier/dp2p',                            'app/cashier/dp2p',                            'default',    'DP2P'],
```
Comment (negarn) [inline scripts/config/pages.js]: could you place this on line 5 so we follow the alphabetical ordering in `cashier/`?

File: src/javascript/app/base/binary_pages.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const Cashier                 = require('../pages/cashier/cashier');
 const DepositWithdraw         = require('../pages/cashier/deposit_withdraw');
 const PaymentAgentList        = require('../pages/cashier/payment_agent_list');
 const PaymentAgentWithdraw    = require('../pages/cashier/payment_agent_withdraw');
+const DP2P                    = require('../pages/cashier/dp2p');
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: same here, should be on line 10

File: src/javascript/app/base/binary_pages.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ const pages_config = {
     forex                    : { module: GetStarted.Forex },
     forwardws                : { module: DepositWithdraw,            is_authenticated: true, only_real: true },
     home                     : { module: Home,                       not_authenticated: true },
-    dp2p                     : { module: DP2P,                       not_authenticated: true },
+    dp2p                     : { module: DP2P,                       is_authenticated: true },
```
Comment (negarn) [inline src/javascript/app/base/binary_pages.js]: same here, should be on line 85

File: src/javascript/app/pages/cashier/dp2p.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
+const ReactDOM     = require('react-dom');
+const React        = require('react');
+const urlForStatic = require('../../../_common/url').urlForStatic;
+const Client       = require('../../base/client');
```
Comment (negarn) [inline src/javascript/app/pages/cashier/dp2p.js:3]:
```suggestion
const React       = require('react');
const ReactDOM     = require('react-dom');
const Client       = require('../../base/client');
const urlForStatic = require('../../../_common/url').urlForStatic;
```

File: src/javascript/app/pages/cashier/dp2p.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            import('@deriv/p2p').then((module) => {
+                const el_dp2p_container = document.getElementById('binary-dp2p');
+                const shadowed_el_dp2p = el_dp2p_container.attachShadow({ mode: 'closed' });
+
+                // const el_head = document.querySelector('head');
+                // const el_main_css = document.createElement('link');
+                // el_main_css.href = urlForStatic('css/p2p.min.css');
+                // el_main_css.rel = 'stylesheet';
+                // el_main_css.type = 'text/css';
+                // el_dp2p_container.innerHTML = module.default;
```
Comment (negarn) [inline src/javascript/app/pages/cashier/dp2p.js]: do we need these comments?

Comment (mahdiaryayi-fs) [reply src/javascript/app/pages/cashier/dp2p.js]: @mahdipourziaei-binary knows that!

Comment (msamprz) [reply src/javascript/app/pages/cashier/dp2p.js]: @mahdiaryayi-fs nope, we don't anymore.

**Review by negarn — APPROVED** [review]

File: src/javascript/app/pages/cashier/dp2p.js (hunk 5 above)
Comment (mahdiaryayi-fs) [reply src/javascript/app/pages/cashier/dp2p.js]: OK. I removed them

**Pull request merged**